### PR TITLE
made header, transparent

### PR DESCRIPTION
**Pull request by eastend-street**
- made menu include about, team, contact, press link

### CODE REVIEW

File: src/components/commons/Header/Header.scss [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
-    height: 10vh;
-}
\ No newline at end of file
+  background-color: transparent;
+  height: 4rem;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+
+  a {
```
Comment (Guntzel00) [inline src/components/commons/Header/Header.scss]: Please use BEM Style

File: src/components/commons/Header/Header.scss [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+    text-decoration: none;
+    color: #fff;
+    transition: 0.5s;
+    &:hover{
+        opacity: 0.7;
+    }
+  }
+
+  .menu {
```
Comment (Guntzel00) [inline src/components/commons/Header/Header.scss]: Please use BEM Style

File: src/components/commons/Header/Header.scss (hunk 1 above)
Comment (eastend-street) [reply src/components/commons/Header/Header.scss]: oh sorry i will fix it !

File: src/components/Header/Header.scss [hunk 3]
```diff
@@ -1,4 +1,99 @@
...
+
 .header {
-    background-color: transparent;
-    height: 10vh;
-}
\ No newline at end of file
+  position: sticky;
+  top: 0;
+  left: 0;
+  background-color: #000;
```
Comment (Guntzel00) [inline src/components/Header/Header.scss]: We will probably need opacity to make the transparent effect

**Pull request merged**